src/web/lib/hakuneko/frontend@classic-dark/jobs.html

```diff
@@ -132,6 +168,13 @@
                 this.ipc.on( 'close', this.onClose.bind( this ) );
             }
 
+            /**
+             *
+             */
+            isQueueEmpty(length) {
```

Comment:
The method name is misleading, i would expect a bool as return type, but it returns the string representation of a related CSS class.